.azure-piplines/master.yml

```diff
@@ -1,35 +1,36 @@
 trigger:
-    - master
+  - master
 
 pr: none
 
 stages:
-    - template: code_quality_checks.yml
-    - stage: "test"
-      displayName: "Python tests"
+  - template: code_quality_checks.yml
+  - stage: "Create package and test"
+    displayName: "Python tests"
 
-      jobs:
-          - job: "python_test"
-            displayName: "Python tests"
-            timeoutInMinutes: 20
-            pool:
-                vmImage: "ubuntu-18.04"
-            steps:
-                - checkout: self
-                  submodules: true
-                - bash: echo "##vso[task.prependpath]$CONDA/bin"
-                  displayName: "add conda to PATH"
-                - bash: |
-                      set -ex
-                      conda --version
-                      conda config --set always_yes yes --set changeps1 no
-                      conda env create -f scipp-widgets.yml
-                  displayName: "create anaconda environment"
-                - bash: |
-                      source activate scipp-widgets
-                      python -m pytest --junitxml=junit/unit-test.xml
-                  displayName: "run pytest"
-          - task: PublishTestResults@2
-            inputs:
-                testResultsFiles: "junit/*.xml"
-            condition: succeededOrFailed()
+    jobs:
+      - job: "linux"
+        displayName: "Build package and test"
+        timeoutInMinutes: 20
+        pool:
+          vmImage: "ubuntu-18.04"
+        steps:
+          - checkout: self
+            submodules: true
+          - bash: echo "##vso[task.prependpath]$CONDA/bin"
+            displayName: "add conda to PATH"
+          - bash: |
+              set -ex
+              conda --version
+              conda install --yes anaconda-client conda-build
+              conda config --set always_yes yes --set changeps1 no
+              conda env create -f scipp-widgets.yml
```

Comment:
You create the environment, but you don't use it. Probably best not to use the environment at all for this job.